src/transformStateWithClones.js

```diff
@@ -1,13 +1,42 @@
 'use strict';
 
 /**
- * @param {Object} state
+ * @param {Object} stateCopy
  * @param {Object[]} actions
  *
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultArray = [];
```

Comment:
`resultArray` is not so good name for a variable, the name should describe a variable, for example, `statesHistory` or something similar